src/identity/components/Identity.tsx

```diff
@@ -2,15 +2,21 @@ import type { IdentityReact } from '../types';
 import { IdentityProvider } from './IdentityProvider';
 import { IdentityLayout } from './IdentityLayout';
 import { useCallback } from 'react';
+import { useOnchainKit } from '../../useOnchainKit';
```

Comment:
I really need to add Linting rule to help organize those imports automatically.

@roushou is something you can help with?

we have those rules from BOAT project https://github.com/coinbase/build-onchain-apps/blob/main/template/web/.eslintrc.js#L135 that will be great to reuse with Biome.

Reply:
Will have a look tomorrow. 3am here :sleeping: